op-chain-ops/deployer/state/intent.go

```diff
@@ -26,6 +26,16 @@ type Intent struct {
 	Chains []*ChainIntent `json:"chains" toml:"chains"`
 
 	GlobalDeployOverrides map[string]any `json:"globalDeployOverrides" toml:"globalDeployOverrides"`
+
+	BaseFeeVaultRecipient common.Address `json:"baseFeeVaultRecipient" toml:"baseFeeVaultRecipient"`
+
+	L1FeeVaultRecipient common.Address `json:"l1FeeVaultRecipient" toml:"l1FeeVaultRecipient"`
+
+	SequencerFeeVaultRecipient common.Address `json:"sequencerFeeVaultRecipient" toml:"sequencerFeeVaultRecipient"`
+
+	Eip1559Denominator uint64 `json:"eip1559Denominator" toml:"eip1559Denominator"`
+
+	Eip1559Elasticity uint64 `json:"eip1559Elasticity" toml:"eip1559Elasticity"`
```

Comment:
Related to above comment, these would get moved into the `ChainIntent` struct since these values will vary per chain

Reply:
Changed in: https://github.com/ethereum-optimism/optimism/pull/12404/commits/2d3aa8a87c5131f22242dc85bd8209a4ba47a81d